docs/querying/operators.md

```diff
@@ -114,6 +114,37 @@ Operations between vectors attempt to find a matching element in the right-hand
 vector for each entry in the left-hand side. There are two basic types of
 matching behavior: One-to-one and many-to-one/one-to-many.
 
+### Vector matching keywords
+
+These vector matching keywords allow for matching between series with different label sets
+providing:
+
+* `on`
+* `ignoring`
+
+Label lists provided to matching keywords will determine how vectors are combined. Examples
+can be found in [One-to-one vector matches](#one-to-one-vector-matches) and in
+[Many-to-one and one-to-many vector matches](#many-to-one-and-one-to-many-vector-matches)
+
+### Group modifiers
+
+These group modifiers enable many-to-one/one-to-many vector matching:
+
+* `group_left`
+* `group_right`
+
+Label lists can be provided to the group modifier contains additional labels from
```

Comment:
Grammar seems off here. Maybe "... can be provided to the group modifier **and contain** additional labels... " or "... can be provided to the group modifier **to specify** additional labels... "?

Reply:
Updated.